website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Permissions 
+
+When a user does not have permissions, hide the related actions, navigation items and views.
+
+### Hide/show relevant UI
+
+In this example, the user's role is **viewer**, and they cannot deploy or destroy a cluster. An explanation is unnecessary as users do not require additional functionality for their day-to-day tasks.
+
+![Image showing two dropdown with lists open, one with only one menu item that is reader specific, while the other has more options that allow for further capabilities.](/assets/patterns/disabled-patterns/permissions-example.png)
+
+Similarly, if a user has the permissions to update their software, show the relevant UI for them to do just that.
```

Comment:
This looks like a new addition? I'm a bit unclear on what this means, are referring to a user being able to change their role?

Reply:
Because updating was part of the old documentation, I wanted to add it into the permissions section.

Ultimately yes, if the user has the proper permissions to update what ever it is, we will show them the relevant message to do just that.

Reply:
I guess what's unclear to me is specifically "update their software", that seems pretty vague. Is there a term or phrase that we can use instead that is more clear?

Reply:
Hmmmm... not entirely sure. Maybe more so "if a user has permissions to update..." maybe thats enough? Removing the "your software?"

Reply:
Yeah, I think removing software helps, maybe also changing to "upgrade" from "update"

Reply:
Resolving, updating jargon.